metrics/config_observability.go

```diff
@@ -69,6 +69,9 @@ type ObservabilityConfig struct {
 	// OpenCensus. "None" disables all backends.
 	RequestMetricsBackend string
 
+	// RequestMetricsReportingPeriodSeconds specifies the request metrics reporting period in sec at queue proxy, eg 1
+	RequestMetricsReportingPeriodSeconds int
```

Comment:
Should this be defaulted in the `defaultConfig` or is 0 going to be default & valid value?

Reply:
I know that this is an implementation detail but the reporting period in the metricsConfig has the following definition:

```
	// reportingPeriod specifies the interval between reporting aggregated views.
	// If duration is less than or equal to zero, it enables the default behavior.
	reportingPeriod time.Duration
```
So the same concept applies here. If a zero value is passed the default underlying behavior is triggered. Probably need to add a comment here.

Reply:
I added a comment to clarify default behavior.

Reply:
Added a default value.